app/(main)/league/[leagueId]/entry/[entryId]/week/WeekTeams.tsx

```diff
@@ -37,26 +37,27 @@ const formatDateTime = (dateString: string): string => {
  * @param props The parameters for the weekly picks page.
  * @param props.field The form field.
  * @param props.schedule The schedule for the week.
+ * @param props.selectedTeams The user's selected teams.
  * @param props.userPick The user's pick.
  * @param props.onWeeklyPickChange The function to call when the user's pick changes.
  * @returns The rendered weekly picks page.
  */
 const WeekTeams = ({
   field,
   schedule,
+  selectedTeams,
   userPick,
   onWeeklyPickChange,
 }: IWeekTeamsProps): JSX.Element => (
   <>
     {schedule.map((scheduledGame) => (
       <RadioGroup
-        onValueChange={field.onChange}
+        onValueChange={(value: string) => onWeeklyPickChange(value)}
```

Comment:
Is this really not a `ChangeEvent<HTMLInputElement>` anymore?

Reply:
I'm curious about this too. I anticipate it would be a ChangeEvent because we are dealing with the event itself instead of only the string. Thoughts? :)

Reply:
Move this radio group outside the loop. There should only be 1 radio group because the user can only make 1 selection from all of these radio options.